app/routes/bdb/components/BdbPage.tsx

```diff
@@ -139,15 +140,20 @@ const BdbPage = () => {
     },
   ];
 
-  const title = 'Bedriftsdatabase';
+  const title = (
+    <Flex gap="var(--spacing-sm)" alignItems="center">
+      <h1>BDB</h1>
+      <Tag className={styles.badge} tag="PRO" color="gray" />
+    </Flex>
+  );
 
   return (
     <Page
       title={title}
       actionButtons={<LinkButton href="/bdb/add">Ny bedrift</LinkButton>}
       tabs={<BdbTabs />}
     >
-      <Helmet title={title} />
+      <Helmet title="BDB" />
```

Comment:
```suggestion
      <Helmet title="Bedriftsdatabase" />
```

Reply:
It's "BDB PRO" now, no longer "Bedriftsdatabase"